psycopg/setup.py

```diff
@@ -41,6 +41,7 @@
         "pproxy ~= 2.7.8",
         "pytest ~= 6.2.5",
         "pytest-asyncio ~= 0.16.0",
+        "pytest-cov",
```

Comment:
Let's pin the version like for the other packages:
```suggestion
        "pytest-cov ~= 3.0.0",
```